pkg/cloud/gen/main.go

```diff
@@ -309,8 +309,19 @@ func (m *Mock{{.Service}}Obj) ToGA() *{{.GA.FQObjectType}} {
 	}
 }
 
+// callOperationRequiresID returns true if the ServiceInfo.Object is
+// of a particular type.
+func callOperationRequiresID(obj string) bool {
+	return strings.HasSuffix(obj, "Route") ||
+		strings.HasSuffix(obj, "Mesh") ||
+		strings.HasSuffix(obj, "tlsRoute") ||
```

Comment:
Do we need HasSuffix or can do a full string match?

My concern is that this may inadvertently cause types to be included that we weren't aware of.

Reply:
in this case this is only for Routes, we have different type (tcp, tlc etc) but we can list them all

Reply:
changed